superagi/worker.py

```diff
@@ -34,6 +34,10 @@
         'task': 'initialize-schedule-agent',
         'schedule': timedelta(minutes=5),
     },
+    'execute_waiting_workflows': {
+        'task': 'execute_waiting_workflows',
```

Comment:
The task 'execute_waiting_workflows' is scheduled to run every 2 minutes. Please ensure the execution time of this task is less than 2 minutes to prevent overlapping runs.

Reply:
Yes as per use case delay pf wait block will be of mostly longer time